src/library/FauxControl/FauxControl.js

```diff
@@ -37,22 +37,20 @@ type Props = {
   variant?: 'success' | 'warning' | 'danger'
 };
 
+// prettier-ignore
 export const componentTheme = (baseTheme: Object) => ({
-  FauxControl_backgroundColor: baseTheme.backgroundColor_input,
+  FauxControl_backgroundColor: baseTheme.input_backgroundColor,
   FauxControl_borderColor: baseTheme.borderColor,
   FauxControl_borderColor_active: baseTheme.borderColor,
   FauxControl_borderColor_focus: baseTheme.borderColor,
-  FauxControl_borderColor_hover: baseTheme.borderColor_hover,
+  FauxControl_borderColor_hover: baseTheme.borderColor_theme_hover,
```

Comment:
This seems to change the `Theme Variable` list display to dark text instead of white; is that an affect of our color contrast library?

![screen shot 2018-04-12 at 11 36 01 am](https://user-images.githubusercontent.com/10079657/38694085-f61cc3de-3e45-11e8-962a-053150838808.png)

Reply:
Yep. We'll look into that in a future story.